cmd/cluster-agent/app/compliance_cmd.go

```diff
@@ -26,17 +23,11 @@ var (
 )
 
 func init() {
-	checkCmd := app.CheckCmd()
-	checkCmd.PreRunE = func(cmd *cobra.Command, args []string) error {
-		// we'll search for a config file named `datadog-cluster.yaml`
-		config.Datadog.SetConfigName("datadog-cluster")
-		err := common.SetupConfig(confPath)
-		if err != nil {
-			return fmt.Errorf("unable to set up global cluster agent configuration: %w", err)
-		}
-		return nil
-	}
+	bundleParams := core.BundleParams{
```

Comment:
Is this correct ? Maybe we should wait for another command to be migrated in the cluster agent so that we can copy the logic here @juliogreff

Reply:
That's all that's necessary as far as I can tell so far, from the experience if migrating other commands 👍

Reply:
Great news !